packages/ethereum-contracts/contracts/gov/SuperfluidGovernanceBase.sol

```diff
@@ -63,12 +63,21 @@ abstract contract SuperfluidGovernanceBase is ISuperfluidGovernance
     {
         if (hostNewLogic != address(0)) {
             UUPSProxiable(address(host)).updateCode(hostNewLogic);
+            UUPSProxiable(address(hostNewLogic)).castrate();
```

Comment:
I paused to think if we could brick the host upgrade by embedding the `castrate()` step. I don't think I can find a case this can be a problem.

If @d10r you think it's not a problem neither, I think we are good to go.

Reply:
no, I can't think of any such case